lib/banks/c6.ex

```diff
@@ -0,0 +1,58 @@
+defmodule BRAN.Banks.C6 do
+  alias BRAN.DigitCalculator
+
+  import BRAN.Banks.Util
+
+  @moduledoc """
+    Documentation `BRAN.Banks.C6`
+  """
+
+  @account_number_size 7
+
+  @weigths [8, 7, 6, 5, 4, 3, 2]
+
+  @doc """
+  Returns a tuple, after checking if the account_number and digit is valid
+
+  ##Examples
+    iex> BRAN.Banks.C6.validate([0,0,0,1], [1,7,9,2,7,0,6], 4)
+    :true
```

Comment:
O exemplo não condiz com o retorno do código

```suggestion
    {:ok, :valid}
```

Reply:
Copy/Paste :(

Reply:
Deixei passar esse ponto! Muito bem observado @log4b ... o que vocês acham de não nesse PR obviamente.. mas como essa é uma lib nossa, habilitarmos os `doctests`?